README.md

```diff
@@ -61,36 +61,38 @@ exports.task = function* (ctx) {
 
 ## Overview
 
-`egg-schedule` is for time-based scheduling, not interval-based scheduling.  
+`egg-schedule` supports both time-based scheduling and interval-based scheduling.
 
-Just put your jobs at `{app_root}/app/schedule`, one file as one job.  
-
-Job file should export `schedule` and `task` properties.
+To setup a schedule task, simply create a job file in `{app_root}/app/schedule`. A file contains one job and should export `schedule` and `task` properties. The rule of thumbs is one job per file.
 
 ## Task
-Task should be a generator function, and accept ctx as params.
+
+Task is a generator function, and accept one parameter, `ctx`. ex: `exports.task = function* (ctx) { ... };`
 
 You can find schedule log at log file, which act like worker request, and contain:
+
 - ctx.method: `SCHEDULE`
 - ctx.path: `/__schedules/${schedulePath}`
 - ctx.query: `scheule config(type=worker&cron=*%2F5%20*%20*%20*%20*%20*)`
```

Comment:
`path=${s}&${qs.stringify(schedule.schedule)}`
should change to `path=<schedule file name>&<schedule config>` ?